src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +150,1078 @@ metadata_pkg.(
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `projection`: Returns the projection matrix, which has size `(p, p_out)`), where
+   `p` and `p_out` are the number of features of the input and ouput respectively.
+
+# Report
+
+The fields of `report(mach)` are:
+`outdim = min(n, indim, maxoutdim)`, where `n` is the
+   number of observations and `indim` the input dimension.
+
+- `indim`: The input dimensions.
+- `outdim`: `min(n, indim, maxoutdim)`, where `n` is the number of observations.
+- `tprincipalvar`: Total variance of the principal components.
+- `tresidualvar`: Total residual variance.
+- `tvar`: Total observation variance (principal + residual variance).
+- `mean`: The mean vector (of length `d`).
+- `principalvars`: The variance of the principal components.
+
+# Examples
+
+```
+using MLJ
+
+PCA = @load PCA pkg=MultivariateStats
+
+X, y = @load_iris
+
+model = PCA(maxoutdim=2)
+mach = machine(model, X) |> fit!
+
+Xproj = transform(mach, X)
+```
+
+See also
+[`KernelPCA`](@ref), [`ICA`](@ref), [`FactorAnalysis`](@ref), [`PPCA`](@ref)
+"""
+PCA
+"""
+$(MMI.doc_header(KernelPCA))
+
+`KernelPCA` Kernel principal component analysis. Using a kernel, the linear
+operations of PCA are performed in a [reproducing Hilbert space](https://en.wikipedia.org/wiki/Reproducing_kernel_Hilbert_space).
+
+# Training data
+
+In MLJ or MLJBase, bind an instance `model` to data with
+
+    mach = machine(model, X)
+
+Where
+
+- `X`: is any table of input features (eg, a `DataFrame`) whose columns
+  are of scitype `Continuous`; check the scitypes with `schema(X)`
+
+Train the machine using `fit!(mach, rows=...)`.
+
+# Hyper-parameters
+
+- `maxoutdim=0`: Controls the the dimension (number of columns) of the output,
+   `outdim`. Specifically,  `outdim = min(n, indim, maxoutdim)`, where `n` is the
+   number of observations and `indim` the input dimension.
+- `kernel::Function=(x,y)->x'y`: The kernel function, takes in 2 vector arguments
+   x and y, returns a scalar value. Defaults to the dot product of `x` and `y`.
+- `solver::Symbol=:auto`: solver to use for the eigenvalues, one of `:eig`(default, uses `LinearAlgebra.eigen`),
+  `:eigs`(uses `Arpack.eigs`).
+- `inverse::Bool=true`: perform calculations needed for inverse transform
+- `beta::Real=1.0`: strength of the ridge regression that learns the inverse transform
+  when inverse is true.
+- `tol::Real=0.0`: Convergence tolerance for eigenvalue solver.
+- `maxiter::Int=300`: maximum number of iterations for eigenvalue solver.
+
+# Operations
+
+- `transform(mach, Xnew)`: Return a lower dimentional projection of the input `Xnew` having the same scitype as `X` above.
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `projection`: Returns the projection matrix, which has size `(p, p_out)`), where
+   `p` and `p_out` are the number of features of the input and ouput respectively.
```

Comment:
For consistency with above (my bad)

```suggestion
- `projection`: Returns the projection matrix, which has size `(indim, outdid)`), where
   `indim` and `outdim` are the number of features of the input and ouput respectively.
```
```suggestion
- `projection`: Returns the projection matrix, which has size `(p, p_out)`), where
   `p` and `p_out` are the number of features of the input and ouput respectively.
```